cmd/containerd-shim-runhcs-v1/delete.go

```diff
@@ -82,6 +82,13 @@ The delete command will be executed in the container's bundle as its cwd.
 			}
 		}
 
+		// hcsshim shim writes panic logs in the bundle directory in a file named "panic.log"
+		// log those messages (if any) on stderr so that it shows up in containerd's log.
+		logs, err := ioutil.ReadFile(filepath.Join(bundleFlag, "panic.log"))
+		if err == nil && len(logs) > 0 {
+			logrus.Errorf("found shim panic logs during delete: %s", string(logs))
```

Comment:
Can we put `logs` as a field (e.g. `logrus.WithField("panicLog", string(logs).Error("found shim panic logs during delete")`

Reply:
Added.